cloud/pkg/synccontroller/synccontroller.go

```diff
@@ -222,6 +240,58 @@ func (sctl *SyncController) manageObjectSync(syncs []*v1alpha1.ObjectSync) {
 		}
 	}
 }
+func (sctl *SyncController) DeleteOutdatedObjectSync() {
+	for {
```

Comment:
plz add an init-delete when sync controller starts, node may be deleted when cloudcore crashes.

Reply:
When the cloudcore start, do one round check and then triggered by node delete event?

Reply:
It depends on which action we pick when the env is not clean, directly exit 1 or try some clean things to see if it can run as expected. Could we unify this logic?